tasks/zadanie_14.php

```diff
@@ -43,7 +43,36 @@
 
 $products = $params[0]; // tej linijki nie ruszamy :)
 
+$lp = 0;
+$sum = 0;
 
-echo "Lp. 1 | Laptop | 1 szt. | 3000 zł | VAT 23% | brutto: 3000 zł\n";
-echo "Lp. 2 | Monitor | 2 szt. | 700 zł | VAT 23% | brutto: 1400 zł\n";
-echo "\nSUMA BRUTTO: 4400 zł";
\ No newline at end of file
+
+foreach ($products as $key => $value){
+	
+    if (is_array($value)) {
+    	 
+    	 	$name = $value['name'];
+        	$qty = $value['qty'];
+        	$vat = $value['vat'];
+        	$price = $value['price'];
+        	
+        	
+       $amount	= $qty * $price ;
+       $sum = $sum + $amount;
+        
+        	$amount	= $qty * $price ;
+        	$sum = $sum + $amount;
+        	
+            if ($key2 == "name") {
+                $name = $value2;
+            }elseif ($key2 == "price") {
+                $price = $value2;
+            }elseif ($key2 == "qty") {
+                $qty = $value2;
+            }elseif ($key2 == "vat") {
+                $vat =$value2;
+            }
+        
+        } echo "Lp. " . ++$lp . " | " . $name . " | " . $qty . " szt. | " . $price . " zł | VAT " . $vat . "% | brutto: " . $amount . " zł" . PHP_EOL;
+    }
+    } echo PHP_EOL . "SUMA BRUTTO: " . $sum . " zł";
```

Comment:
> PHP Parse error:  Unmatched '}' in /home/runner/work/shoper-up/shoper-up/tasks/zadanie_14.php on line 78
```suggestion
    echo PHP_EOL . "SUMA BRUTTO: " . $sum . " zł";
```

Tutaj masz jedna za dużo klamrę ;)